tracr/compiler/craft_graph_to_model.py

```diff
@@ -27,38 +27,24 @@
 NodeID = nodes.NodeID
 
 
-def _get_longest_path_length_to_node(graph: nx.DiGraph, sources: Sequence[Node],
-                                     node: Node) -> int:
-  """Returns the lengths of the longest path from sources to node.
+def compute_computational_depth(graph, sources):
+    computational_depth = {}
 
-  Only SOps count towards the length of a path.
+    def dfs(node, depth):
+        if isinstance(graph.nodes[node][nodes.EXPR], rasp.Select):
```

Comment:
Is this to not count selects in the path? I think you want to check "not ininstance(..., rasp.SOp)" instead

Reply:
Also I think it's slightly cleaner to have this check at the recursive call to dfs instead here